### ISSUE TITLE
VM: Non inclusive loop limits

### ISSUE THREAD

**Issue by ejfitzgerald** [fetchai/ledger#911]
The following case should not print anything

```
for (i in 0:0)
  Print("A-TEST: Wish we didn't see this - " + toString(i));
endfor
```

**Comment by ejfitzgerald** [comment]
Notes: Disable support for decrementing for the time being

**Comment by wkaluza** [comment]
Fixed in b3afd9cacc60d906749ea0f404fa7cf727197533

**Comment by wkaluza** [comment]
Fixed in b3afd9cacc60d906749ea0f404fa7cf727197533
